test/db/esil/x86_32

```diff
@@ -2370,7 +2370,9 @@ CMDS=<<EOF
 # Common parts
 #
 e asm.arch=x86
-e asm.assembler=x86.as
+?==x86==`e asm.arch`
```

Comment:
this looks awful :(

```
?== x86 `e asm.arch`
``` 
Looks better imo. Probably the `==` were just done to make the parsing easy.

Reply:
?== x y implies that no spaces are supported. i can change for that. and i bet that would work with the new parser. any hint how to do that?

Reply:
or do you want to handle this when the new command gets accepted, reviewed and tested?

Reply:
updated

Reply:
Yeah, I think it looks much better. New shell will be able to deal with spaces as well just by doing
```
?== "first string" "second string"
```